autogen.sh

```diff
@@ -64,7 +73,11 @@ done
 echo
 echo "=== generating configure files in main directory ==="
 autoreconf -vif
-echo "=== done === "
+if test "$?" = "0" ; then
+    echo "=== done === "
+else
+    echo "=== failed === "
```

Comment:
Should it exit here when failure instead of keep going?

Reply:
Fixed.